### PR TITLE
R4 Appointment reasonCode Patch Enhancement

### PR DESCRIPTION
**Pull request by ldumler1996**
Screen Shot 2020-06-26 at 10 32 44 AM
Screen Shot 2020-06-26 at 10 32 13 AM
Screen Shot 2020-06-26 at 10 32 22 AM

Description
----
Updated documentation for Appointment reasonCode Patch

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

File: content/millennium/r4/scheduling/appointment.md [hunk 1]
```diff
@@ -231,6 +231,34 @@ The `ETag` response header indicates the current `If-Match` version to use on su
...
+<%= headers status: 200 %>
+<pre class="terminal">
+Cache-Control: no-cache
+Content-Length: 0
+Content-Type: text/html
+Date: Tue, 26 Mar 2019 15:42:29 GMT
+Etag: W/"1"
+Last-Modified: Tue, 26 Mar 2019 15:42:27 GMT
+Vary: Origin
+X-Request-Id: 47306a14c8a2c3afd4ab85aa9594101d
```
Comment (ldumler1996) [inline content/millennium/r4/scheduling/appointment.md:255]: Need to capture a sandbox 200 response and update X-Request-Id and Last-Modified.

Comment (MaxPhilips) [reply content/millennium/r4/scheduling/appointment.md:255]: conveniently we put a note on here indicating that examples are non-normative, so we can get away with making up pretend x-request-id and last-modified values :)

Comment (ldumler1996) [reply content/millennium/r4/scheduling/appointment.md:255]: Are you implying that we don't need to update this information, then?

File: lib/resources/r4/appointment_patch.yaml [hunk 2]
```diff
@@ -40,6 +40,26 @@ operations:
...
+        "path": "/reasonCode",
+        "value": [
+          {
+            "text": "I have a headache"
+          }
+        ]
+      }
+    note: >
+      <ul>
+        <li>This patch operation is only supported when updating the Appointment.status with a value of "booked"</li>
```
Comment (MaxPhilips) [inline lib/resources/r4/appointment_patch.yaml:61]: to match language already present in the file

```
When present, this patch operation must be accompanied by the replace Status patch operation with a value of "booked"
```

File: lib/resources/r4/appointment_patch.yaml [hunk 3]
```diff
@@ -40,6 +40,26 @@ operations:
...
+      </ul>
```
Comment (MaxPhilips) [inline lib/resources/r4/appointment_patch.yaml:62]: add note indicating only one codeable concept is allowed

```
Only one CodeableConcept may be provided in the value array
```

Comment (MaxPhilips) [inline lib/resources/r4/appointment_patch.yaml:62]: add note indicating reasonCodes are freetext
```
CodeableConcept.text is required and CodeableConcept.coding, if present, will be ignored
```

File: content/millennium/r4/scheduling/appointment.md (hunk 1 above)
Comment (MaxPhilips) [reply content/millennium/r4/scheduling/appointment.md:255]: correct, we do not need to update those two fields

File: lib/resources/r4/appointment_patch.yaml (hunk 2 above)
Comment (ldumler1996) [reply lib/resources/r4/appointment_patch.yaml]: [Fixed](https://github.com/cerner/fhir.cerner.com/commit/3de61110223359030a6c5e9135ab15199eb6aff4#diff-07dbf4d3264a1e7a98f8d854b77f88e9R61)

File: lib/resources/r4/appointment_patch.yaml (hunk 3 above)
Comment (ldumler1996) [reply lib/resources/r4/appointment_patch.yaml:64]: [Added](https://github.com/cerner/fhir.cerner.com/commit/3de61110223359030a6c5e9135ab15199eb6aff4#diff-07dbf4d3264a1e7a98f8d854b77f88e9R63)

Comment (ldumler1996) [reply lib/resources/r4/appointment_patch.yaml:64]: [Added and fixed a typo](https://github.com/cerner/fhir.cerner.com/commit/3d1e971d35ff233b93fb577f25faa807df709823#diff-07dbf4d3264a1e7a98f8d854b77f88e9R62)

**Comment by MaxPhilips** [comment]
live verification

Screen Shot 2020-07-01 at 5 49 25 PM

**Pull request merged**